default.nix

```diff
@@ -14,6 +14,6 @@ let
     ];
   };
   packages = pkgSet.config.hsPkgs;
-in packages.cachix.components.exes.cachix  // {
+in packages.cachix.components.all  // {
```

Comment:
I suppose this isn't public api.

Reply:
Yes it is: https://input-output-hk.github.io/haskell.nix/reference/library/#component

Reply:
Oh I meant cachix `/default.nix`

Reply:
Kind of - need to fix readme I guess.